### ISSUE TITLE
Small bug - sentry (build 754)

### ISSUE THREAD

**Issue by dapper27** [Kemmey/Kemmey-TeslaWatch-Public#369]
Just updated to 754. With sentry mode on, watch shows 'Sentry mode active' on main page underneath battery level, phone app doesn't. Phone app does show sentry mode toggle as active in the controls page.

**Comment by Kemmey** [comment]
Are you by any chance showing the big battery on your phone and the split view on watch? Big battery doesn't show the labels. 

Possibly it should?

**Comment by dapper27** [comment]
Yes I must have pressed the battery by accident and changed the display mode at some point. Oops!
